Source/Libraries/CorruptCore/IRealTime.cs

```diff
@@ -8,7 +8,6 @@ namespace RTCV.CorruptCore
 
     public interface IRealTime
     {
-        event EventHandler<RealTimeEventArgs> Step;
```

Comment:
This builds but I don't have context for this.

Reply:
Some plugins subscribe to it, none that are in public repos in the moment though.

Reply:
Can I rename it to satisfy `CA1716` and break the private plugins? Or should I make an exception here?

Reply:
They're just not done really. All the other changes already completely broke them all so yeah go ahead. I've long accepted that lots of stuff is going to need fixes as you refactor. It's much less effort to recompile them than it is to deal with a messy codebase so go feel free to go forwards.

Reply:
Done

Reply:
Ah, this is actually being used in Bizhawk. I'll need to make a companion fix here.